test/k8s/updates.go

```diff
@@ -48,6 +48,8 @@ var _ = Describe("K8sUpdates", func() {
 	)
 
 	BeforeAll(func() {
+
+		SkipIfIntegration(helpers.CIIntegrationAKS)
```

Comment:
This test is upgrading to cilium dev, so skipping this as well. Need confirmation